region/lowernorfair/west/Screw Attack Room.json

```diff
@@ -1270,33 +2072,124 @@
           ]
         }
       ],
-      "note": "Diagonally shinespark towards the middle door.",
-      "devNote": [
-        "Obstacle C is indicating that Samus has a shinecharge ready to be used.",
-        "Useful only if you cannot reach the door at 2."
+      "note": "Diagonally shinespark towards the middle door."
```

Comment:
You climb up and horizontal spark here.

Reply:
Good catch, looks like the note got copy-pasted by mistake, removed it now